test/unit/minify.ts

```diff
@@ -48,4 +48,17 @@ describe('minify tool', () => {
     assert.deepEqual(objectBefore, objectAfter);
     assert.deepEqual(objectBefore, parsedObjectAfter);
   });
+
+  it('minifies the proto js file', async () => {
+    const echoProtoJsFixture = path.join(fixturesDir, 'echo.js');
+    const echoProtoJs = path.join(testDir, 'echo.js');
+    await fsp.copyFile(echoProtoJsFixture, echoProtoJs);
+    const statBefore = await fsp.stat(echoProtoJs);
+    const resultBefore = require(echoProtoJs);
+    await minify.main(testDir);
+    const statAfter = await fsp.stat(echoProtoJs);
+    const resultAfter = require(echoProtoJs);
```

Comment:
Here I'm wondering if the second require of the same filename will just take it from cache. Maybe either invalidate the cache (https://stackoverflow.com/questions/9210542/node-js-require-cache-possible-to-invalidate) or just copy it into another file just to be sure.

Reply:
Would you also want to change this in the test above? I just copied your pattern for the previous test: https://github.com/googleapis/gax-nodejs/blob/4d59f35fed63c43e4679b636de964d2c81a6609e/test/unit/minify.ts#L41